mk/spksrc.service.start-stop-status

```diff
@@ -35,11 +35,17 @@ start_daemon ()
             else
                 SVC_CD="cd ${SVC_CWD}; "
             fi
+            if [ -n "${SYNOPKG_DSM_VERSION_MAJOR}" ] && [ "$SYNOPKG_DSM_VERSION_MAJOR" -gt 7 ]; then
```

Comment:
Shouldn't this be `-ge 7` ?

Reply:
Good catch. Now I wonder how any of this worked....

To make it consistent with the other ifs I reversed it:

```sh
if [ -n "${SYNOPKG_DSM_VERSION_MAJOR}" ] && [ "$SYNOPKG_DSM_VERSION_MAJOR" -lt 7 ]; then
    SU="su ${EFF_USER} -s" 
else
    SU=""
fi
```